docs/guides/v9-upgrade-guide.md

```diff
@@ -12,19 +12,53 @@ Please read this guide carefully if you plan to update your app to this release.
 
 We recommend upgrading your application for each major version gradually, e.g. if you plan to update from 7.x to 9.x, then migrate first to 8.x and then to version 9.
 
-### Removal of deprecated packages and components
+The tables below show what will be removed and what are they replaced with. We also marked if there are [codemods](#ui-codemods) available.
 
-The following deprecated components are removed in v9:
+##### Table of Contents
```

Comment:
Merged the 2 guides like we agreed before (with v8 deprecations guide). Also "pimped it up" a bit and started a "deprecated prop values" block as well.

Reply:
Nice, I like it